webapp/tests/test_render_datalib.py

```diff
@@ -287,22 +255,9 @@ def _build_requestContext(self, startTime=datetime(1970, 1, 1, 0, 0, 0, 0, pytz.
             'tzinfo': tzinfo
         }
 
-    def test_nonempty_true(self):
-      values = range(0,100)
-      series = TimeSeries("collectd.test-db.load.value", 0, len(values), 1, values)
-      self.assertTrue(nonempty(series))
-
-    def test_nonempty_false_empty(self):
-      series = TimeSeries("collectd.test-db.load.value", 0, 1, 1, [])
-      self.assertFalse(nonempty(series))
-
-    def test_nonempty_false_nones(self):
-      series = TimeSeries("collectd.test-db.load.value", 0, 4, 1, [None, None, None, None])
-      self.assertFalse(nonempty(series))
-
     def test__merge_results(self):
       pathExpr = 'collectd.test-db.load.value'
-      startTime=datetime(1970, 1, 1, 0, 10, 0, 0, pytz.timezone(settings.TIME_ZONE)),
+      startTime=datetime(1970, 1, 1, 0, 10, 0, 0, pytz.timezone(settings.TIME_ZONE))
```

Comment:
This is a bugfix, previously startTime was getting defined as a tuple because of the trailing comma.  It didn't directly affect these test but became obvious in `test_prefetchData`